pandas/core/arrays/base.py

```diff
@@ -36,7 +36,7 @@ class ExtensionArray:
     """
     Abstract base class for custom 1-D array types.
 
-    pandas will recognize instances of this class as proper arrays
+    pandas will recognize instances of this claspandas.api.extensions.ExtensionArray.views as proper arrays
```

Comment:
besides the line too long, looks like something went wrong here, we need ```:class:`pandas.api...```

Reply:
This was just a control-f gone wrong. It's removed below.